shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/Rules/Utxo.hs

```diff
@@ -199,8 +200,8 @@ instance
 --   the mint field.
 consumed ::
   forall era.
-  ( UsesValue era,
-    UsesTxOut era,
+  ( Val.Val (Core.Value era),
+    HasField "value" (Core.TxOut era) (Core.Value era),
     HasField "certs" (Core.TxBody era) (StrictSeq (DCert (Crypto era))),
     HasField "inputs" (Core.TxBody era) (Set (TxIn (Crypto era))),
     HasField "mint" (Core.TxBody era) (Core.Value era),
```

Comment:
consumed is way over constrained, it does not need the UsesValue or UsesTxout constraints.